build.gradle

```diff
@@ -216,6 +215,19 @@ jar {
     from sourceSets.generated.output
 }
 
+jib {
+    extraDirectories.paths = ['build/gen-expanded']
+    to {
+        image = "gcr.io/broad-jade-dev/jade-data-repo"
+        tags = [(System.env.GCR_TAG ?: "latest")]
+    }
+}
+task setupGeneratedDir(type: Copy) {
```

Comment:
Maybe add a comment explaining why this task it needed?

Reply:
added